tests/visualization_tests/test_pareto_front.py

```diff
@@ -56,6 +56,20 @@ def create_study_2d() -> Study:
     return study
 
 
+def create_study_2d_with_constraints(
+    constraints_func: Callable[[FrozenTrial], Sequence[float]]
+) -> Study:
+    sampler = optuna.samplers.TPESampler(seed=0, constraints_func=constraints_func)
+    study = optuna.create_study(directions=["minimize", "minimize"], sampler=sampler)
+
+    study.enqueue_trial({"x": 1, "y": 2})
+    study.enqueue_trial({"x": 1, "y": 1})
+    study.enqueue_trial({"x": 0, "y": 2})
+    study.enqueue_trial({"x": 1, "y": 0})
+    study.optimize(lambda t: [t.suggest_int("x", 0, 2), t.suggest_int("y", 0, 2)], n_trials=4)
+    return study
```

Comment:
Is it possible to merge this function to `create_study_2d` by making `constraints_func` optional?